packages/teleport/src/teleportContext.tsx

```diff
@@ -54,6 +54,10 @@ class TeleportContext implements types.Context {
     return this.storeUser.isSso() === false;
   }
 
+  isAccessRequestEnabled() {
```

Comment:
Lets refactor TeleportContext by moving all feature flags to a dedicated method

```tsx
getFeatureFlags() {
  const userContext = this.storeUser;
  return {
    account: userContext.isSso() === false,
    audit: userContext.getEventAccess().list,
    authConnector: storeUser.getConnectorAccess().list,
    roles: userContext.getRoleAccess().list,
    trustedClusters: storeUser.getTrustedClusterAccess().list,
    users: userContext.getUserAccess().list,
    applications: userContext.getAppServerAccess().list,
  }
}
```

And then use `workflows` to name a feature instead of RequestAccess